upload/lambdas/api_server/validation_scheduler.py

```diff
@@ -23,6 +25,10 @@ def __init__(self, uploaded_file: UploadedFile):
         self.file_key = self.file.upload_area.uuid + '/' + urllib.parse.quote(self.file.name)
         self.config = UploadConfig()
 
+    def check_file_can_be_validated(self):
+        if self.file.size >= 1000000000000:
```

Comment:
Include a comment here about why we are using this specific byte number. In this case, it is because we have a 1tb volume limit for staging files from s3 during validation process

Reply:
👍